### PR TITLE
Count if e test

### PR DESCRIPTION
**Pull request by kaatish**
This PR adds multi-gpu testing for count_if_e primitive.

### CODE REVIEW

**Review by seunghwak — APPROVED** [review]
Looks great!

**Comment by seunghwak** [comment]
rerun tests

**Comment by kaatish** [comment]
rerun tests

**Comment by ChuckHastings** [comment]
rerun tests

File: cpp/tests/prims/mg_count_if_e.cu [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+      hr_clock.stop(&elapsed_time);
+      std::cout << "MG count if e took " << elapsed_time * 1e-6 << " s.\n";
+    }
+
+    //// 4. compare SG & MG results
+
+    if (prims_usecase.check_correctness) {
+      cugraph::graph_t<vertex_t, edge_t, weight_t, store_transposed, false> sg_graph(handle);
+      std::tie(sg_graph, std::ignore) =
+        input_usecase.template construct_graph<vertex_t, edge_t, weight_t, store_transposed, false>(
```
Comment (ChuckHastings) [inline cpp/tests/prims/mg_count_if_e.cu:261]: I just changed this in a PR that got merged.  If you merge the latest 21.10 you'll find this doesn't compile.

You need to change this call to something like:
`cugraph::test::construct_graph<vertex_t, edge_t, weight_t, store_transposed, false>(handle, input_usecase, true, false)`

File: cpp/tests/prims/mg_count_if_e.cu [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+
+    // 2. create MG graph
+
+    if (cugraph::test::g_perf) {
+      CUDA_TRY(cudaDeviceSynchronize());  // for consistent performance measurement
+      handle.get_comms().barrier();
+      hr_clock.start();
+    }
+    auto [mg_graph, d_mg_renumber_map_labels] =
+      input_usecase.template construct_graph<vertex_t, edge_t, weight_t, store_transposed, true>(
```
Comment (ChuckHastings) [inline cpp/tests/prims/mg_count_if_e.cu:208]: I just changed this in a PR that got merged. If you merge the latest 21.10 you'll find this doesn't compile.

This call should be something like:
`cugraph::test::construct_graph<vertex_t, edge_t, weight_t, store_transposed, true>(handle, input_usecase, prims_usecase.test_weighted, true);`

**Review by ChuckHastings — CHANGES_REQUESTED** [review]

**Comment by ChuckHastings** [comment]
rerun tests

**Review by ChuckHastings — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**